internal/dtr/in_test.go

```diff
@@ -59,4 +59,23 @@ var _ = Describe("In", func() {
 			}))
 		})
 	})
+
+	Context("empty/no-op configuration", func() {
+		It("should not fail", func() {
+			_, err := In(feed(Config{}))
+
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("should return given version", func() {
+			version := Version{"ref": "foobar"}
+			result, _ := In(feed(Config{
```

Comment:
Second parameter is an error, right? We should run an Expect against that.

Reply:
added